admin/sql/CreateTables.sql

```diff
@@ -852,6 +852,24 @@ CREATE TABLE gender ( -- replicate
     gid                 uuid NOT NULL
 );
 
+CREATE TABLE genre ( -- replicate (verbose)
+    id                  SERIAL, -- PK
+    name                VARCHAR NOT NULL,
+    comment             VARCHAR(255) NOT NULL DEFAULT '',
+    gid                 uuid NOT NULL
+);
+
+CREATE TABLE genre_alias ( -- replicate (verbose)
```

Comment:
Shouldn't we have a `unique_primary_genre_alias` function (and associated trigger) too? (See the other `unique_primary_*_alias` functions in admin/sql/CreateFunctions.sql.)